homeassistant/components/tile/device_tracker.py

```diff
@@ -132,12 +133,10 @@ def source_type(self) -> str:
     @callback
     def _handle_coordinator_update(self) -> None:
         """Respond to a DataUpdateCoordinator update."""
-        self._update_from_latest_data()
-        self.async_write_ha_state()
+        self.async_schedule_update_ha_state(force_refresh=True)
```

Comment:
This is not correct. See https://github.com/home-assistant/core/pull/56650#discussion_r716102623